sourceview.py

```diff
@@ -52,7 +56,7 @@ def __init__(self, parent=None):
         # override style settings to the same font and size
         # (python lexer has styles 0 ... 15)
         for i in range(16):
-          self.SendScintilla(QsciScintilla.SCI_STYLESETFONT, i, fontName)
+          #self.SendScintilla(QsciScintilla.SCI_STYLESETFONT, i, fontName)
```

Comment:
why disable this?

Reply:
Some problem with Scintillia, not anymore -> enabled again.